tests/test_gossip.py

```diff
@@ -938,7 +938,7 @@ def test_gossip_addresses(node_factory, bitcoind):
         {'type': 'ipv4', 'address': '127.0.0.1', 'port': 9735},
         {'type': 'ipv6', 'address': '::', 'port': 3},
         {'type': 'ipv6', 'address': '::', 'port': 9735},
```

Comment:
```suggestion
        {'type': 'ipv6', 'address': '::', 'port': 19846},
```
Not sure if it is the correct port number, but this causes the failure in the CI

Reply:
I have changed these now. But they were OK as they were set to 9375 manually and setting port to anything still works.